tests/pipelines/unclip/test_unclip.py

```diff
@@ -385,15 +417,19 @@ def test_unclip_karlo(self):
         generator = torch.Generator(device="cpu").manual_seed(0)
         output = pipeline(
             "horse",
-            num_images_per_prompt=1,
             generator=generator,
             output_type="np",
         )
 
-        image = output.images[0]
+        image = np.asarray(pipeline.numpy_to_pil(output.images)[0], dtype=np.float32)
+        expected_image = np.asarray(pipeline.numpy_to_pil(expected_image)[0], dtype=np.float32)
+
+        # Karlo is extremely likely to strongly deviate depending on which hardware is used
+        # Here we just check that the image doesn't deviate more than 10 pixels from the reference image on average
+        avg_diff = np.abs(image - expected_image).mean()
 
+        assert avg_diff < 10, f"Error image deviates {avg_diff} pixels on average"
```

Comment:
Pretty high avg difference, but it doesn't seem like we can get closer than this.